src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashPullRequestsBuilder.java

```diff
@@ -18,12 +18,15 @@
     private StashRepository repository;
     private StashBuilds builds;
 
-    public static StashPullRequestsBuilder getBuilder() {
-        return new StashPullRequestsBuilder();
-    }
+    public StashPullRequestsBuilder(AbstractProject project, StashBuildTrigger trigger) {
+        if (project == null || trigger == null) {
```

Comment:
Can you change it to `Objects.requireNonNull()`?

Reply:
Sure, good idea. The trigger argument can be annotated as non-null, it's always called with `this`.

In the long term, we should probably eliminate StashPullRequiestBuilder, it's a thin "appendix" to StashBuildTrigger with 1-1 relationship. But that would be another PR.